BaragonAgentService/src/main/java/com/hubspot/baragon/agent/resources/RequestResource.java

```diff
@@ -182,7 +182,12 @@ public Response revert(@PathParam("requestId") String requestId) throws Interrup
       LOG.info(String.format("Reverting to %s", update));
       
       try {
-        configHelper.apply(update, Optional.<String>absent(), false);
+        configHelper.apply(update, Optional.<BaragonService>absent(), false);
+      } catch (MissingTemplateException e) {
+        // If this service previously didn't exist, no config was written, doesn't matter that we can't find the template
+        if (!maybeOldService.isPresent() || !maybeOldService.get().getLoadBalancerGroups().contains(loadBalancerConfiguration.getName())) {
+          return Response.ok().build();
```

Comment:
not sure this is a safe call to make -- we shouldn't be returning what BaragonService will interpret as a success when we experienced a failure. also, wont the `else` case of this just fall through to the `Response.ok()` below?